tests/e2e/test_linker_run.py

```diff
@@ -26,15 +25,15 @@
     [
         # slurm
         (
-            "pipeline.yaml",
-            "input_data.yaml",
-            "environment_slurm.yaml",
+            "e2e/pipeline.yaml",
```

Comment:
I just moved a few specification files around to be consistent with organization I'd done elsewhere